{"uuid":"6cf6b954-69ba-4ef2-a438-30d740791d4a","historyId":"c5f1fb5c0fa7870b8bb48ff591cf3c44","fullName":"com.SauceDemoProject.TestScripts.TC_RegistrationPage_002.userRegistration","labels":[{"name":"package","value":"com.SauceDemoProject.TestScripts.TC_RegistrationPage_002"},{"name":"testClass","value":"com.SauceDemoProject.TestScripts.TC_RegistrationPage_002"},{"name":"testMethod","value":"userRegistration"},{"name":"parentSuite","value":"ParallelTestSuite"},{"name":"suite","value":"Chrome Test : 126 : Windows 11"},{"name":"subSuite","value":"com.SauceDemoProject.TestScripts.TC_RegistrationPage_002"},{"name":"host","value":"ECCINHYD04681"},{"name":"thread","value":"24316@ECCINHYD04681.TestNG-test=Chrome Test : 126 : Windows 11-2(34)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"severity","value":"critical"}],"links":[],"name":"Verify User Registration","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. \nHost info: host: 'ECCINHYD04681', ip: '192.168.246.243'","trace":"org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. \nHost info: host: 'ECCINHYD04681', ip: '192.168.246.243'\r\n\tat org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:536)\r\n\tat org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:232)\r\n\tat org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:161)\r\n\tat org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:141)\r\n\tat com.SauceDemoProject.Utilities.BaseClass.setUp(BaseClass.java:158)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)\r\n\tat org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)\r\n\tat org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)\r\n\tat org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\r\n\tat java.base/java.lang.Thread.run(Thread.java:1570)\r\nCaused by: org.openqa.selenium.TimeoutException: java.net.http.HttpConnectTimeoutException: HTTP connect timed out\nBuild info: version: '4.21.0', revision: '79ed462ef4'\nSystem info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '22.0.1'\nDriver info: driver.version: RemoteWebDriver\r\n\tat org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:389)\r\n\tat org.openqa.selenium.remote.http.AddSeleniumUserAgent.lambda$apply$0(AddSeleniumUserAgent.java:42)\r\n\tat org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)\r\n\tat org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute(JdkHttpClient.java:355)\r\n\tat org.openqa.selenium.remote.tracing.TracedHttpClient.execute(TracedHttpClient.java:54)\r\n\tat org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:89)\r\n\tat org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:75)\r\n\tat org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:61)\r\n\tat org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:162)\r\n\tat org.openqa.selenium.remote.TracedCommandExecutor.execute(TracedCommandExecutor.java:51)\r\n\tat org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:518)\r\n\t... 15 more\r\nCaused by: java.net.http.HttpConnectTimeoutException: HTTP connect timed out\r\n\tat java.net.http/jdk.internal.net.http.MultiExchange.toTimeoutException(MultiExchange.java:620)\r\n\tat java.net.http/jdk.internal.net.http.MultiExchange.getExceptionalCF(MultiExchange.java:565)\r\n\tat java.net.http/jdk.internal.net.http.MultiExchange.lambda$responseAsyncImpl$7(MultiExchange.java:485)\r\n\tat java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)\r\n\tat java.base/java.util.concurrent.CompletableFuture$UniHandle.tryFire(CompletableFuture.java:911)\r\n\tat java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)\r\n\tat java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2194)\r\n\tat java.net.http/jdk.internal.net.http.Exchange.checkCancelled(Exchange.java:330)\r\n\tat java.net.http/jdk.internal.net.http.Exchange.cancel(Exchange.java:293)\r\n\tat java.net.http/jdk.internal.net.http.MultiExchange.cancel(MultiExchange.java:266)\r\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection$ConnectTimerEvent.handle(PlainHttpConnection.java:103)\r\n\tat java.net.http/jdk.internal.net.http.HttpClientImpl.purgeTimeoutsAndReturnNextDeadline(HttpClientImpl.java:1788)\r\n\tat java.net.http/jdk.internal.net.http.HttpClientImpl$SelectorManager.run(HttpClientImpl.java:1385)\r\nCaused by: java.net.ConnectException: HTTP connect timed out\r\n\tat java.net.http/jdk.internal.net.http.MultiExchange.toTimeoutException(MultiExchange.java:621)\r\n\t... 12 more\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"version","value":"126"},{"name":"resolution","value":"1920x1080"},{"name":"platform","value":"Windows 11"},{"name":"browser","value":"chrome"}],"start":1720012342169,"stop":1720012342171}